index.js

```diff
@@ -28,8 +28,8 @@ export default {
         android: nullFunc
     }),
     deleteAllItems: Platform.select({
-       ios: SpotlightSearch?.deleteAllItems,
-       android:  nullFunc
+        ios: () => SpotlightSearch?.deleteAllItems(),
```

Comment:
I'm not sure if the `?` should be there anymore - I'm not certain that is valid to invoke a function with optional chaining. In which case, this could be written as `SpotlightSearch && SpotlightSearch.deleteAlIitems()

Reply:
@jdmunro Would you like me to change optional chaining everywhere to the old school object check?)

Reply:
Just in this one case here is fine! 👍 
Thanks!

Reply:
I would like to change across the file to have a consistent code because it can confuse other people why is there an optional chaining check and in another place just via &&.
What do you think?

Reply:
My mistake, I thought this wouldn't be valid (invoking a function via optional chaining). Since it appears to be valid, there's nothing to change.